# Generated by Django 4.0.5 on 2022-08-13 19:04

from django.db import migrations, models

import users.validators


class Migration(migrations.Migration):

    dependencies = [
        ("users", "0002_alter_customuser_email"),
    ]

    operations = [
        migrations.AlterField(
            model_name="customuser",
            name="email",
            field=models.EmailField(
                max_length=254, unique=True, verbose_name="Email address"
            ),
        ),
        migrations.AlterField(
            model_name="customuser",
            name="group",
            field=models.CharField(max_length=10, unique=True),
        ),
        migrations.AlterField(
            model_name="customuser",
            name="username",
            field=models.CharField(
                error_messages={
                    "unique": ("A user with that username already exists.",)
                },
                max_length=15,
                unique=True,
                validators=[users.validators.UnicodeUsernameValidator()],
            ),
        ),
    ]
